highlighters/main/main-highlighter.zsh

```diff
@@ -432,6 +449,56 @@ _zsh_highlight_highlighter_main_paint()
       fi
     fi
 
+    # Expand aliases.
+    # TODO: this should be done iteratively, e.g., 'alias x=y y=z z=w\n x' 
```

Comment:
Trailing whitespace here.

Reply:
Mind if we fix this on master post merge? Along with the other ~10 instances in master?

Reply:
Works for me.